documentation/src/docs/python.md

```diff
@@ -0,0 +1,130 @@
+# Python API for the connector
+
+This Pravega connector of Python API provides a data source and data sink for Flink streaming jobs.
+
+Your Flink streaming jobs could use Pravega as their storage with these [Python API Wrappers](https://github.com/pravega/flink-connectors/tree/master/src/main/python).
```

Comment:
Describe a bit here, saying it is basically a python wrapper of the java implementation of `FlinkPravegaReader`, `FlinkPravegaWriter` as well as some related class using py4j. Meanwhile, we should guide users to read the java api doc first, so we don't need to explain the same thing again in this document, like the `default_scope` and `streamcut`